[TT-14598] Pump 1.14.0 RNs.mdx (#1444)
TykTechnologies/tyk-docs: developer-support/release-notes/pump.mdx

---

## 1.14 Release Notes

### 1.14.0 Release Notes

> #### Release Date xx 2026

Reviewer: ```suggestion
#### Release Date 06 March 2026
```

| [Valkey](https://valkey.io/download/)                     | 8.0.x, 8.1.x      | 7.2.x, 8.0.x, 8.1.x      |                            

Given the time difference between your upgrade and the release of this version, we recommend customers verify the ongoing support of third-party dependencies they install, as their status may have changed since the release.

#### Deprecations

There are no deprecations in this release.

#### Upgrade instructions

For users currently on v1.13.3, we strongly recommend promptly upgrading to the latest release. 
Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade Instructions.

#### Downloads
- [Docker Image v1.14.0](https://hub.docker.com/r/tykio/tyk-pump-docker-pub/tags?page=&page_size=&ordering=&name=v1.14.0)
  - ```bash
    docker pull tykio/tyk-pump-docker-pub:v1.14.0
    ```
- Source code tarball for OSS - [GH Tyk Pump Repo](https://github.com/TykTechnologies/tyk-pump/releases/tag/v1.14.0)

#### Changelog
<a id="Changelog-v1.14.0" data-scroll-offset></a>

##### Added

<AccordionGroup>

<Accordion title='Enhance Pump Security with Custom CA Certificates'>

> You can now establish fully trusted and secure TLS connections from Tyk Pump to Kafka, Elasticsearch, and Splunk services that use custom or internally-issued SSL certificates. 

Reviewer 2: ```suggestion
You can now establish fully trusted and secure TLS connections from Tyk Pump to Kafka, Elasticsearch, and Splunk services that use certificates signed by specific Certificate Authorities.
```

> To configure this, set the CA file path using the `ssl_ca_file` option or the `TYK_PMP_PUMPS_<PUMP_NAME>_META_SSLCAFILE` environment variable

Reviewer 2: ```suggestion
To configure this, set the path to the PEM file containing the CA certificate using the `ssl_ca_file` option or the `TYK_PMP_PUMPS_<PUMP_NAME>_META_SSLCAFILE` environment variable.
```

---

## 1.14 Release Notes

### 1.14.0 Release Notes

#### Release Date xx 2026

#### Release Highlights

> In this release, we've enhanced the security capabilities of Tyk Pump by introducing custom CA certificate support for Kafka, Elasticsearch, and Splunk pumps. For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v1.14.0). 

Reviewer: ```suggestion
In this release, we've enhanced Tyk Pump's security capabilities by introducing support for custom CA certificates for Kafka, Elasticsearch, and Splunk pumps. For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v1.14.0). 
```

| [Redis](https://redis.io/download/)                       | 6.x - 7.0         | 6.x - 7.x                | 
| [Valkey](https://valkey.io/download/)                     | 8.0.x, 8.1.x      | 7.2.x, 8.0.x, 8.1.x      |                            

Given the time difference between your upgrade and the release of this version, we recommend customers verify the ongoing support of third-party dependencies they install, as their status may have changed since the release.

#### Deprecations

There are no deprecations in this release.

#### Upgrade instructions

For users currently on v1.13.3, we strongly recommend promptly upgrading to the latest release. 
Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade Instructions.

#### Downloads
- [Docker Image v1.14.0](https://hub.docker.com/r/tykio/tyk-pump-docker-pub/tags?page=&page_size=&ordering=&name=v1.14.0)
  - ```bash
    docker pull tykio/tyk-pump-docker-pub:v1.14.0
    ```
- Source code tarball for OSS - [GH Tyk Pump Repo](https://github.com/TykTechnologies/tyk-pump/releases/tag/v1.14.0)

#### Changelog
<a id="Changelog-v1.14.0" data-scroll-offset></a>

##### Added

> <AccordionGroup>
> 
> <Accordion title='Enhance Pump Security with Custom CA Certificates'>
> You can now establish fully trusted and secure TLS connections from Tyk Pump to Kafka, Elasticsearch, and Splunk services that use custom or internally-issued SSL certificates. To configure this, set the CA file path using the `ssl_ca_file` option or the `TYK_PMP_PUMPS_<PUMP_NAME>_META_SSLCAFILE` environment variable
> </Accordion>

Reviewer: ```suggestion
<Accordion title='Enhance Pump Security with Custom CA Certificates'>
You can now establish fully trusted and secure TLS connections from Tyk Pump to Kafka, Elasticsearch, and Splunk services that use custom or internally-issued SSL certificates. 

To configure this, set the CA file path using the `ssl_ca_file` option or the `TYK_PMP_PUMPS_<PUMP_NAME>_META_SSLCAFILE` environment variable
</Accordion>
```
